warehouse/templates/manage/manage_base.html

```diff
@@ -34,6 +34,18 @@
     </span>
   </div>
   {% endif %}
+
+  {% if request.user.two_factor_allowed and not request.user.has_two_factor %}
+  <div class="notification-bar notification-bar--danger">
+    <span class="notification-bar__message">
+      <i class="fa fa-exclamation-triangle" aria-hidden="true"><span class="sr-only">Warning</span></i>
+      Two factor authentication is disabled.
+      <a href="{{ request.route_path('manage.account', _anchor="two-factor") }}">
+        Add a second factor.
```

Comment:
I think this warning might be a little too scary/prominent. I'm concerned about a few things:
* users who don't understand what two factor authentication is being confused that something went wrong when logging in, or when they're not on the account management page;
* users who don't want to enable 2FA being unable to dismiss this;
* general fatigue/disregard for these warnings.

I do think we should "announce" this to users somehow, though. I think if we changed this to a dismissable yellow warning that said something like "Two-factor authentication is available, [enable it now for your account](url)." I'd feel better about this.

Reply:
> I do think we should "announce" this to users somehow, though. I think if we changed this to a dismissable yellow warning that said something like "Two-factor authentication is available, [enable it now for your account](url)." I'd feel better about this.

This sounds good to me! I'll move it from `manage_base.html` to `account.html` as well, so that users will only see it during account management.

Reply:
Kept it in `manage_base.html` per off-thread discussion.